c7n/resources/ssm.py

```diff
@@ -597,3 +598,117 @@ def register_resource(cls, registry, resource_class):
 
 
 resources.subscribe(PostItem.register_resource)
+
+
+@resources.register('ssm-document')
+class SSMDocument(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'ssm'
+        enum_spec = ('list_documents', 'DocumentIdentifiers', None)
```

Comment:
we should be defaulting to Owner=Self there's no governance constraint on first (amazon) or third party documents as their not manageable from the account the policy is executing in.

Reply:
Got it -- I just made this change.